tests/cicd_utils/test_scripts/test_generate_internal_api_rst.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""Tests for generate_internal_api_rst.py script."""
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+
+from generate_internal_api_rst import generate_module_rst, organize_modules
+
+# Mueve esta línea después de las importaciones
+SCRIPT_PATH = Path(__file__).parents[3] / "cicd_utils/cicd/scripts/generate_internal_api_rst.py"
+sys.path.append(str(SCRIPT_PATH.parent))
```

Comment:
Btw you don't need this. You can just import from the cicd package. (Eg, https://github.com/tpvasconcelos/ridgeplot/blob/ed13afc8ca499ddde5a4b19753a5672115552f33/tests/cicd_utils/test_scripts/test_extract_latest_release_notes.py#L8 )